File: TP1/TP1 python/python fede/ejercicio 2/Producto.py
class Producto:
    
    def __init__(self, articulo, dosis, unidadDosis, precio): #constructor de la clase
        self.articulo = articulo
        self.dosis = dosis
        self.unidadDosis = unidadDosis
        self.precio = precio
        
    def getPrecio(self):
        return self.precio
    
    def imprimir(self):
        print("{}       {}{}       {}".format(self.articulo, self.dosis, self.unidadDosis, self.precio))

File: Practica Parcial/Cesar/Parcial 2023/Parcial 23/src/Producto.java
public class Producto {
    private String articulo;
    private Integer dosis;
    private Integer precio;
    
    
    public Producto(String articulo, Integer dosis, Integer precio) {
        this.articulo = articulo;
        this.dosis = dosis;
        this.precio = precio;
    }
    public Integer getPrecio() {
        return precio;
    }
    public String toString() {
        return articulo+"            "+ dosis+"mg"+"            " + precio+"\n";
    }
}
